vercel.json

```diff
@@ -1,33 +1,21 @@
 {
-  "buildCommand": "npm run build  > >(tee -a build-output.log) 2> >(tee -a build-output.log >&2) && ! (cat build-output.log | grep -qi error) && echo \"No errors found in build output\"",
+  "buildCommand": "npm run build:vercel:dist && npm run build:vercel:functions",
```

Comment:
Unless there's a good reason to separate these out, I'd keep just a single script